### PR TITLE
Add option to disable reviews  obligation on pull request

### PR DESCRIPTION
**Pull request by QM-OSchwab**
Used mainly for projects with only one developer.

### CODE REVIEW

**Comment by johndgiese** [comment]
Thank you for your contribution!

File: rdm/backends/github_issue.py [hunk 1]
```diff
@@ -197,7 +198,7 @@ def change_approvals(pull_request):
     '''
     external_review = 'external-review' in [l.name for l in pull_request.labels]
 
-    if external_review:
+    if external_review or (system['reviews_required'] == 'false'):
```
[the remark this answers is not in the document]
Comment (QM-OSchwab) [reply rdm/backends/github_issue.py]: To be honest, I did not see any example of using boolean from system.yml elsewhere in rdm code, so I presumed that everything was considered as string.

File: rdm/init_files/data/system.yml [hunk 2]
```diff
@@ -72,3 +72,6 @@ current_srs_document_id: SRS-001
 
 # The document ID of the "Software Design Specification" (the How)
 current_sds_document_id: SDS-001
+
+# are pull requests reviews mandatory
+reviews_required: true
```
[the remark this answers is not in the document]
Comment (QM-OSchwab) [reply rdm/init_files/data/system.yml]: _When you realize you're better in Jamaican than in english_ :)

**Comment by QM-OSchwab** [comment]
Sorry  for loss of time, David. I'm reviewing the problems. I don't get why I don't have any error about  _system_ missing when I test here.

**Comment by johndgiese** [comment]
@QM-OSchwab no worries at all! Let me know if there is anything I can help with.

**Pull request closed without merging**